diff -Naur qt-everywhere-opensource-src-4.8.6.orig/configure qt-everywhere-opensource-src-4.8.6/configure
--- qt-everywhere-opensource-src-4.8.6.orig/configure	2014-04-10 20:37:08.000000000 +0200
+++ qt-everywhere-opensource-src-4.8.6/configure	2017-04-30 18:03:40.130326093 +0200
@@ -7729,6 +7729,9 @@
     *-g++*)
 	# Check gcc's version
 	case "$(${QMAKE_CONF_COMPILER} -dumpversion)" in
+	    [56]*)
+		QMakeVar add QMAKE_CXXFLAGS -std=gnu++98
+		;;
 	    4*|3.4*)
 		;;
             3.3*)
